lib/capistrano/recipes/deploy/strategy/jenkins_artifact.rb

```diff
@@ -50,11 +50,16 @@ def _compression_type_to_switch(type)
   def deploy!
     dir_name = exists?(:is_multibranch_job) && fetch(:is_multibranch_job) ? fetch(:branch) : fetch(:build_project)
 
+    release_name_from = fetch(:release_name_from, 'build_at')
+    if release_name_from != 'build_at' && release_name_from != 'deploy_at' then
```

Comment:
You can omit `then`.
- http://ruby-doc.com/docs/ProgrammingRuby/html/tut_expressions.html#UH